1.0/docs/devguide/tests.md

```diff
@@ -0,0 +1,367 @@
+    extension for Selenium. See the 
+    [Web Component Tester Polycast][workaround-example] for 
+    a demonstration.
+
+1.  `cd` to the base directory of the element.
+
+1.  (Optional) Install and save `wct` locally as a bower component so that your 
+    tests can always import the `wct` runtime.
+
+        bower install --save Polymer/web-component-tester
+
+    The global `wct` that you installed earlier actually serves its own 
+    copy of the `wct` runtime (`browser.js`) whenever it encounters any URL
+    that ends with `web-component-tester/browser.js`. Installing it locally
+    is just a precaution. 
+
+1.  Create a directory for the tests. 
+
+        mkdir test
+
+    When you run `wct` with no arguments, it automatically searches for
+    a directory named `test` and runs any tests it finds in there. You
+    can use another name for your directory, but you'll need to specify
+    the path to the directory when you run `wct` (`wct path/to/tests`).
+
+1.  Create a test.
+
+        <!doctype html>
+        <html>
+          <head>
+            <meta charset="utf-8">
+            <script src="../bower_components/webcomponentsjs/webcomponents-lite.js"></script>
+            <script src="../bower_components//web-component-tester/browser.js"></script>
+            <!-- import the element to test -->
+            <link rel="import" href="../seed-element.html">
+          </head>
+          <body>
+            <!-- use the document as a place to set up your fixtures -->
+            <test-fixture id="seed-element-fixture">
+              <template>
+                <seed-element>
+                  <h2>seed-element</h2>
+                </seed-element>
+              </template>
+            </test-fixture>
+            <script>
+              suite('<seed-element>', function() {
+                var myEl;
+                setup(function() {
+                  myEl = fixture('seed-element-fixture');
+                });
+                test('defines the "author" property', function() {
+                  assert.equal(myEl.author.name, 'Dimitri Glazkov');
+                });
+              });
+            </script>
+          </body>
+        </html>
+
+1.  Go to the base directory of your element and run your tests.
+
+        wct
+
+    ![output from successful wct unit test run](/1.0/images/wct-output.png)
+
+    `wct` automatically finds all of the browsers on your system and runs
+    your tests against each one.
+
+## Create stub methods
+
+Stubs enable you to replace default implementations with custom methods. This
+is useful for catching side effects. 
+
+    setup(function() {
+      stub('paper-button', {
+        click: function() {
+          console.log('paper-button.click called');
+        }
+      });
+    });
+
+You don't have to use stubs directly on individual elements. You can override
+the implementation for all elements of a given type. 
+
+## Create stub elements
```

Comment:
:heavy_check_mark: added a paragraph of when / why to use stubs... the sample could be much improved to demonstrate why you would use this, but the new paragraph explains it pretty well